app/routes/__auth/user.server.ts

```diff
@@ -14,8 +17,64 @@ export async function getUser({ headers }: Request) {
   const email = session.get('email') as string
   const groups = session.get('groups') as string[]
   const idp = session.get('idp') as string | null
-
+  const refreshToken = session.get('refreshToken') as string
+  const cognitoUserName = session.get('cognitoUserName') as string
+  const accessToken = session.get('accessToken') as string
   if (!sub) return null
+  const user = {
+    sub,
+    email,
+    groups,
+    idp,
+    refreshToken,
```

Comment:
We may want to not expose the refreshToken in the returned user object, because the application itself will never have a need for it and it is very sensitive.

Reply:
We do use it to call the refresh though, and the user object is what we use to set the session. Any suggestions for how to not include it?

Reply:
Actually I was able to do some rearranging and got it out of the getUser method, let me know what you think